perllib/FixMyStreet/App/Controller/Auth.pm

```diff
@@ -130,6 +132,28 @@ sub sign_in : Private {
     return;
 }
 
+=head 2 throttle_username
+
+Set up memcache to watch for repeated login attempts from the same user name
+within a specified time
+
+=cut
+
+sub throttle_username : Private {
+    my ( $self, $c, $details ) = @_;
+    my $name = $details->{'username'};
+    if (my $conf = $c->cobrand->feature('throttle_username')) {
+        if (Memcached::get($name) ) {
```

Comment:
`my $count = Memcached::increment($name, $conf->{time});`
Would this not increment by the amount of timeout in the setting, rather than increment by 1 attempt?